.fixtures.yml

```diff
@@ -2,10 +2,10 @@ fixtures:
   repositories:
     stdlib:
       repo: git://github.com/puppetlabs/puppetlabs-stdlib
-      ref: '4.6.0'
+      ref: '4.20.0'
```

Comment:
IMO we should unpin the modules here. In general we test against master to find breaking changes before they are released.

Reply:
@bastelfreak works for me, I'll adjust in #242